### ISSUE TITLE
Intercom secure mode

### ISSUE THREAD

**Issue by maartendraijer** [mike-north/ember-intercom-io#45]
I can't find anything configurable regarding to [Intercom's secure mode](https://docs.intercom.io/configure-intercom-for-your-product-or-site/staying-secure/enable-secure-mode-on-your-web-product).

So I'm wondering, is there a way to put in a `user_hash` somewhere?

**Comment by maartendraijer** [comment]
I managed by creating a `{{secure-intercom-io}}` component and providing the `user_hash` to component and then to the `start()` method like this:

```js
// components/secure-intercom-io.js
import Ember from 'ember';

const {
    Component,
    get,
    inject
} = Ember;

export default Component.extend({
    intercom: inject.service(),
    didInsertElement() {
        this._super(...arguments);
        get(this, 'intercom').start({user_hash: this.get('user_hash')});
    },
    willDestroyElement() {
        this._super(...arguments);
        get(this, 'intercom').stop();
    }
});```

**Comment by mike-north** [comment]
I've been using the `user-state` branch, which is not yet in a mergeable state (sorry, I haven't had time yet). Feel free to fork that and use in your app

**Comment by bgentry** [comment]
This was solved by #136. I am using secure mode successfully in my app.

**Comment by rstellar** [comment]
@maartendraijer can you confirm if #136 worked for you and close this?
